ga4gh/frontend_exceptions.py

```diff
@@ -5,6 +5,7 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import exceptions
```

Comment:
https://docs.python.org/2/library/exceptions.html

"The exceptions are defined in the module exceptions. This module never needs to be imported explicitly: the exceptions are provided in the built-in namespace"